lib/Models/ArcGisMapServerCatalogItem.js

```diff
@@ -222,8 +221,12 @@ ArcGisMapServerCatalogItem.prototype._load = function() {
                 // Results of a single layer query. Make it look like a multi layer query result.
                 that.updateFromMetadata(results[0], {"layers": [results[1]]}, results[2], false, results[1]);
             } else {
-                console.error("Unusable ArcGIS Mapserver layers metadata: \n" + layersMetadata);
-                throw new RuntimeError('Unusable ArcGIS Mapserver layers metadata from ' + layersUri.toString() + '. See console log for details.');
+                var message = defined(results[0].error) ? results[0].error.message : 'This dataset returned unusable metadata.';
+                that.terria.error.raiseEvent(new TerriaError({
```

Comment:
`load` should usually return errors via promise rejection, rather than raising `terria.error` and returning success.